api/models/PrintingForm3D.js

```diff
@@ -34,7 +34,7 @@ const PrintingForm3DSchema = new Schema(
 
     // unique id for query
     // id = email of the user who requested
-    id: {
+    email: {
```

Comment:
please rebase your branch. these changes were already merged in

Reply:
@ThaiQ this branch is behind. please rebase it
![image](https://user-images.githubusercontent.com/36345325/71301958-9ca04780-235a-11ea-8608-a9279f65dfd7.png)